src/Symfony/Component/Security/Core/Tests/Authorization/Voter/LegacyAbstractVoterTest.php

```diff
@@ -13,6 +13,16 @@
 
 use Symfony\Component\Security\Core\Authorization\Voter\AbstractVoter;
 
+class LegacyAbstractVoterTest extends AbstractVoterTest
```

Comment:
is this class still needed if we test the legacy case in the ``AbstractVoterTest`` directly ?